slave.mk

```diff
@@ -232,6 +233,9 @@ $(info "INCLUDE_KUBERNETES"              : "$(INCLUDE_KUBERNETES)")
 $(info "TELEMETRY_WRITABLE"              : "$(TELEMETRY_WRITABLE)")
 $(info "ENABLE_SYNCHRONOUS_MODE"         : "$(ENABLE_SYNCHRONOUS_MODE)")
 $(info )
+else
+$(info SONiC Build System for $(CONFIGURED_PLATFORM):$(CONFIGURED_ARCH))
```

Comment:
Why is this line in else not outside if block?

Reply:
I felt it was still useful for SONIC_BUILD_QUIETER to print that it entered SONiC build system for debug purposes.
The information I added to the simple `SONiC Build System` in this line is redundant with what is in the ifdef block above. Since I introduced this one liner I felt it was meaningful to print the targeted platform and arch.
Both `CONFIGURED_PLATFORM` and `CONFIGURED_ARCH` have their dedicated line in the configuration dump above.
I would prefer keeping things as is but I'm open to suggestions.